db/src/db.ts

```diff
@@ -254,7 +254,7 @@ export class DB extends EventEmitter {
           return it.end(resolve);
         }
         const { value } = JSON.parse(rawValue);
-        if (wrappedMatch({ key, value }, filter)) {
+        if (value !== undefined /* Not a tombstone */ && wrappedMatch({ key, value }, filter)) {
```

Comment:
This doesn't belong on this PR, I think.

Reply:
My bad, but if you reviewed it I'm merging.